### DISCUSSION TITLE
Documentation with the current helics functions

### DISCUSSION THREAD

**Discussion by Barsha96** [GMLC-TDC/HELICS#2478] [Q&A] [answered]
Hello Team, 
I am currently integrating helics to use Gridlab-D for my research. I am looking for the documentation for all the functions that are available with latest helics like helicsFederateRegisterGlobalPublication, helicsCreateBroker, helicsCreateFederateInfo and so on. I was thinking if there are any kind of documents where these functionalities are defined and described. If so, it would be a great help in understanding these functions and their parameters. 

Best, 
Barsha

**Comment by trevorhardy** [comment, marked as answer]
For the C-based functions such as what you've listed above, I find the [Python documentation](https://python.helics.org/api/capi-py/) the easiest to navigate and the function names are identical to the [pure C API](https://docs.helics.org/en/latest/references/api-reference/C_API.html).  If you're new to HELICS reading through the [User Guide](https://docs.helics.org/en/latest/user-guide/co-simulation_overview.html) will be helpful and there's also the fairly comprehensive [Configuration Options Reference](https://docs.helics.org/en/latest/references/configuration_options_reference.html) that effectively discusses many of APIs.

Does this answer your question or are there additional details you're interested in?

**Reply by Barsha96** [reply]
I find these helpful. Thank you very much!
